src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected: chosen }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  const active = id => activeTab.id !== id && chosen(id);
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const { id, title } = tab;
```

Comment:
You can do destructing rigth after map( .  Example:  tabs.map(({ id, title }) .  So you won`t need additional return statement and  " {  } "

Reply:
thanks